packages/edit-post/src/components/meta-box-titles/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+
+function MetaBoxTitles( { titles, titleWrapper } ) {
```

Comment:
Actually, I'm not certain what value this component brings, as it seems like we already have the selector as an abstraction to get the meta-boxes titles